common/utils/openebs/upgrade/test.yml

```diff
@@ -0,0 +1,62 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+    - block:
+
+#        - include_tasks: /common/utils/application_liveness_check.yml
+#          when: liveness_label != ''
```

Comment:
@vibhor995 Include this liveness check also.

Reply:
@nsathyaseelan yes.